Core/Utilities/Glue2.py

```diff
@@ -70,7 +70,7 @@ def getGlue2CEInfo(vo, host):
       shareEndpoints = shareInfo['attr'].get('GLUE2ShareEndpointForeignKey', [])
       ceInfo = __getGlue2ShareInfo(host, shareEndpoints, shareInfo['attr'], siteDict[siteName]['CEs'])
       if not ceInfo['OK']:
-        gLogger.error("Could not get CE info for %s:" % shareID, ceInfo['Message'])
+        gLogger.warn("Could not get CE info for %s:" % shareID, ceInfo['Message'])
```

Comment:
As far as I understand, `warn` is still not printed by default in scripts, so all these problems pass silently and one could miss problems with the assumptions about the GLUE2 schema.

Reply:
And I should fix the parsing of the CERN information ASAP ;)

Reply:
you are indeed right:
```

Reply:
To me, this looks odd. WARN, in my own logic, should be printed if the level is 'NOTICE'. @chaen @atsareg was there a discussion on this point in the past?

Reply:
I remember one lengthy discussion that never converged (archives on the lhcb-datamanagement mailing list, title 'log and returned error', started on the 25th of April 2014.. from the top of my head :p )

Reply:
Do you think I should restore these to `error`? Also in light of #3768

Reply:
Yes, I think hat would be better

Reply:
Done.